language/tools/move-unit-test/tests/test_sources/cross_module_aborts.exp

```diff
@@ -8,7 +8,7 @@ Failures in 0x1::B:
 
 ┌── failing_test ──────
 │ error[E11001]: test failure
-│   ┌─ tests/test_sources/cross_module_aborts.move:5:9
```

Comment:
What was the reason for this line being removed btw?

Reply:
In order to test natives, I'm using a native from the vector module. However the path to that module will differ based on where you're running it/if it's in CI or not. So I updated the test runner to strip out everything but the name of the file to prevent that issue from cropping up.